exporter/templates/cat1/r3_1/_code.xml

```diff
@@ -1,5 +1,7 @@
 <{{- valueOrDefault .TagName "code" -}}
-  {{- if hasPreferredCode .PreferredCode}} code='{{- .PreferredCode.Code}}' codeSystem='{{- oidForCodeSystem .PreferredCode.CodeSystem}}' {{.RenderExtraContent}}
+  {{- if hasPreferredCode .PreferredCode}}
+    {{- if .MapDataCriteria.DataCriteria.Negation}} nullFlavor='NA' {{.RenderExtraContent}}{{- else}} code='{{- .PreferredCode.Code}}' codeSystem='{{- oidForCodeSystem .PreferredCode.CodeSystem}}' {{.RenderExtraContent}}
```

Comment:
just realized both are actually legal. we use double in all of the templates so it might be worth changing just for the sake of consistency. I'm see a few other instances of single quotes in this file as well